source/extensions/filters/http/cache/simple_http_cache/simple_http_cache.cc

```diff
@@ -47,7 +47,10 @@ class SimpleLookupContext : public LookupContext {
 class SimpleInsertContext : public InsertContext {
 public:
   SimpleInsertContext(LookupContext& lookup_context, SimpleHttpCache& cache)
-      : key_(dynamic_cast<SimpleLookupContext&>(lookup_context).request().key()), cache_(cache) {}
+      : key_(dynamic_cast<SimpleLookupContext&>(lookup_context).request().key()),
+        entry_vary_headers_(
```

Comment:
entry_vary_headers_(key_.getVaryHeaders())

Reply:
I don't think this works because `key_` is just a `Key` object, it doesn't have access to `getVaryHeaders`.
Unless I'm missing something that I don't know.

Reply:
Ignore--I misread.